### PR TITLE
Add static definitions for Conditions to allow for better linting

### PR DESCRIPTION
**Pull request by markpeek**
Thought I'd address #171 prior to release. 

@michael-k could you review and approve?

### CODE REVIEW

File: scripts/generate_conditions.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "StringLike",
+    "StringNotLike",
+]
+
+_condition_qualifier_strings = ["ForAnyValue", "ForAllValues"]
+
+
+def make_condition(type_name: str, condition_name: str) -> None:
+    print(f"class {type_name}(ConditionElement):")
+    print(f"    condition = '{condition_name}'")
```
Comment (michael-k) [inline scripts/generate_conditions.py:37]: This should also be accepted by black but produces code that doesn't need formatting by black.  Same in line 41.

```suggestion
    print(f'    condition = "{condition_name}"')
```

File: scripts/generate_conditions.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    #    dict(condition=condition_name + "IfExists"),
+    # )
+
+
+# Create condition classes
+for i in _condition_strings:
+    make_condition(i, i)
+
+    for qual in _condition_qualifier_strings:
+        make_condition(qual + i, "%s:%s" % (qual, i))
```
Comment (michael-k) [inline scripts/generate_conditions.py:62]:
```suggestion
        make_condition(f"{qual}{i}", f"{qual}:{i}")
```

File: scripts/generate_conditions.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    # )
+    # globals()[type_name + "IfExists"] = type(
+    #    type_name + "IfExists",
+    #    (ConditionElement,),
+    #    dict(condition=condition_name + "IfExists"),
+    # )
+
+
+# Create condition classes
+for i in _condition_strings:
```
Comment (michael-k) [inline scripts/generate_conditions.py:58]: `i` isn't a good name for a variable.  But since it's only in `scripts/` and existed for a long time :shrug:

**Review by michael-k — APPROVED** [review]
LGTM

A few nitpicks in `scripts/`, but they aren't blocking anything.

**Comment by markpeek** [comment]
Thank you @michael-k for the review. Made changes and committed.

**Pull request merged**